pyproject.toml

```diff
@@ -1,3 +1,14 @@
+[project]
+name = "rocprof_compute"
+requires-python = ">=3.8"
+
+[project.optional-dependencies]
+developer = [
+    "black>=22.6.0",
+    "isort>=5.12.0",
+    "pre-commit",
+]
+
 [tool.black]
 line-length = 90
 target-version = ['py38', 'py39']
```

Comment:
I think it's already covered by requires-python at line 3 but should we target py310 and py311 as well here?